examples/all-clusters-app/linux/BUILD.gn

```diff
@@ -57,7 +64,52 @@ source_set("chip-all-clusters-common") {
   include_dirs =
       [ "${chip_root}/examples/all-clusters-app/all-clusters-common/include" ]
 
-  cflags = [ "-Wconversion" ]
+  if (chip_enable_pw_rpc) {
+    defines = [
+      "PW_RPC_ENABLED",
+      "PW_RPC_ATTRIBUTE_SERVICE=1",
+      "PW_RPC_BUTTON_SERVICE=1",
+      "PW_RPC_DESCRIPTOR_SERVICE=1",
+      "PW_RPC_DEVICE_SERVICE=1",
+      "PW_RPC_LIGHTING_SERVICE=1",
+      "PW_RPC_TRACING_SERVICE=1",
+    ]
+
+    sources += [
+      "${chip_root}/examples/platform/linux/Rpc.cpp",
+      "${chip_root}/examples/platform/linux/system_rpc_server.cc",
+    ]
+
+    deps += [
+      "$dir_pw_hdlc:pw_rpc",
```

Comment:
This is begging for better abstraction (ie, define a common library or at the very least a reusable template())

Reply:
I moved most of the rpc stuff to a new file. Is this OK for you or you had something else in mind?

Reply:
@mspang any feedback?

Reply:
Closing no-feedback since Dec